blockchain/types/src/block_data.rs

```diff
@@ -8,16 +8,20 @@ use serde::{Deserialize, Serialize};
 #[derive(Clone, Deserialize, Eq, Message, PartialEq, Serialize)]
 pub struct BlockData {
     #[prost(message, required, tag = 1)]
-    block: Block,
+    /// The block header.
+    pub block: Block,
```

Comment:
I believe these weren't originally `pub` so that the `BlockData` is immutable. Is there anything non-test-related that require these fields to be directly accessible and modifiable?